src/utils/questions.js

```diff
@@ -0,0 +1,196 @@
+const { getDepartments, getRoles, getEmployee } = require("./choices");
+
+const askMenuQuestions = async(inquirer) => {
+    const questions = [{
+        type: "list",
+        name: "option",
+        message: "Please choose from one of the following options: ",
+        choices: [{
+                name: "Add a Department",
+                value: "addDepartment",
+            },
+            {
+                name: "View All Departments",
+                value: "viewDepartment",
+            },
+            {
+                name: "Add a Role",
+                value: "addRole",
+            },
+            {
+                name: "View All Roles",
+                value: "viewRole",
+            },
+            {
+                name: "Add a Employee",
+                value: "addEmployee",
+            },
+            {
+                name: "View All Employees",
+                value: "viewEmployee",
+            },
+            {
+                name: "Update an Employee's Role",
+                value: "updateEmployeeRole",
+            },
+            {
+                name: "Update an Employee's Manager",
+                value: "updateEmployeeManager",
+            },
+            {
+                name: "View Employees by Department",
+                value: "viewEmployeeDepo",
+            },
+            {
+                name: "View All Managers",
+                value: "viewManager",
+            },
+            {
+                name: "Delete a Department",
+                value: "deleteDepartment",
+            },
+            {
+                name: "Delete a Role",
+                value: "deleteRole",
+            },
+            {
+                name: "Delete a Employee",
+                value: "deleteEmployee",
+            },
+            {
+                name: "Calculate Department Utilized Budget",
+                value: "calcUtilBudget",
+            },
+            {
+                name: "Quit",
+                value: "quit",
+            },
+        ],
+    }, ];
+
+    const answer = await inquirer.prompt(questions);
```

Comment:
Instead of declaring a variable of `answer` and immediately returning it, you can simply just return the whole statement:

`return await inquirer.prompt(questions)`